test/test_StateMachineDevice.py

```diff
@@ -35,6 +35,17 @@ class MockStateMachineDevice(StateMachineDevice):
 
 
 class TestStateMachineDevice(unittest.TestCase):
+    def test_not_implemented_errors(self):
+        self.assertRaises(NotImplementedError, StateMachineDevice)
+
+        with patch('lewis.devices.StateMachineDevice._get_state_handlers',
+                   return_value={'test': Mock()}):
+            self.assertRaises(NotImplementedError, StateMachineDevice)
+
+            with patch('lewis.devices.StateMachineDevice._get_initial_state',
+                       return_value='test'):
+                self.assertRaises(NotImplementedError, StateMachineDevice)
```

Comment:
This test will pass if only `_get_transition_handlers` raises that exception, and `_get_state_handlers` and `_get_initial_state` do nothing (or something else).

It will also pass if there is some other base method that also raises `NotImplementedException` on construction. Perhaps this is fine like that, just pointing it out in case it's not what you intended.